examples/calibrate_test_signal.py

```diff
@@ -0,0 +1,116 @@
+"""
+Capture the signal acquired on the GDS. and converts to analytic 
+================================================================
+A single capture of the signal on the GDS is converted to analytic signal,
+frequency filtered and the absolute is taken to calculate the $V_{amp}$. The
+final plot that is generated plots the raw data, analytic data, filtered
+analytic data, and the absolute value and the caluclated $V_{amp}$ (in mV). 
+This script serves two purposes in calibrating test signal generated by the
+AFG: 
+
+1. Calculating the raw output of the AFG (no attenuator assemblies). For this
+step, simply set the calibrating_atten to False. Note that for the next step
+you need to set the input_Vamp to the value calculated in this step.
+
+2. Calculating the voltage ratio of the attenuator assemblies (placed between
+the AFG output and the GDS). Prior to running the script the user needs to
+have a rough estimate of the expected_Vamp when the attenuator assemblies are
+in place. The calculated voltage ratio and dB are printed on the final
+generated plot.
```

Comment:
not sure if steps are okay in docstring (couldn't find much on google). Feel free to edit but the gist is that first you calculate AFG --> GDS to get the actual V_input (calibrating_atten=False) and then that value is used in calculating the actual voltage ratio (V_input / V_atten)

Reply:
Thinking about this I think this means that the cells in Verify.py showing the AFG-->GDS is unnecessary since it's included in this script.